dags/bq_events_to_amplitude.py

```diff
@@ -98,5 +114,20 @@
         ),
         task_id=devtools_task_id
     )
+    devtools_shredder = gke_command(
+        task_id="devtools_amplitude_shredder",
+        name="shredder-amplitude-devtools",
+        command=[
+            "script/shredder_amplitude",
+            "--date={{ ds }}",
+            "--api-key={{ var.value.amplitude_api_key }}",
+            "--secret-key={{ var.value.amplitude_secret_key }}",
+            "--table-id=moz-fx-data-shared-prod.telemetry.deletion_request_v4",
+            "--user-id-field=client_id",
+        ],
+        docker_image="mozilla/bigquery-etl:latest",
+        dag=dag,
+    )
 
     [wait_for_telemetry_event, wait_for_telemetry_main] >> devtools_export
+    wait_for_telemetry_event >> [fenix_shredder, devtools_shredder]
```

Comment:
It's not clear to me why we need this dependency.

Reply:
because we need to wait for copy_deduplicate before reading from stable tables